hedera-state-validator/src/main/resources/log4j2.xml

```diff
@@ -12,13 +15,22 @@
             </Policies>
             <DefaultRolloverStrategy max="1000"/>
         </RollingFile>
+      <File name="StateAnalysisFileAppender" fileName="state-analysis.log">
+        <PatternLayout pattern="%d [%t] %-5level %logger{36} - %msg%n%throwable"/>
+      </File>
     </Appenders>
     <Loggers>
-        <Logger name="com.hedera.statevalidation.listener.SummaryGeneratingListener" level="INFO">
-            <AppenderRef ref="ConsoleAppender"/>
-        </Logger>
+      <Logger name="com.hedera.statevalidation.listener.SummaryGeneratingListener" level="INFO">
+        <AppenderRef ref="SummaryConsoleAppender"/>
+      </Logger>
+      <Logger name="com.hedera.statevalidation.AnalyzeCommand" level="INFO">
```

Comment:
What about com.hedera.statevalidation.analyzer.StateAnalyzer? It has its own logs

Reply:
com.hedera.statevalidation.analyzer.StateAnalyzer has only debug logs, and for this, there is the following config:
```
        <Root level="DEBUG">
            <AppenderRef ref="FileAppender"/>
        </Root>
```